src/prep_data.R

```diff
@@ -1,20 +1,21 @@
-add_flow_condition <- function(data_in, date_start, date_end){
+
+#' @description Bin percentile data into flow condition categories
```

Comment:
```suggestion
#' @description Bin percentile data (`percentile_bin`) into flow condition (`percentile_cond`) categories
```
I suggest this at a bare minimum to clearly link percentile condition to `percentile_cond` (which also appears in `plot_cartogram.R`).

Personally, I would probably move the assignment of `percentile_cond` into the body of the function to clarify the connection:

```r
@param percentile_cond chr, vector of percentile condition labels. Length must be equal to the length of `breaks` + 1

add_flow_condition <- function(data_in, date_start, date_end, breaks, percentile_cond = c("Driest", "Drier", "Dry", "Normal","Wet","Wetter", "Wettest"))


```